Mention both spritezero and spreet (#785)
maplibre/maplibre-style-spec: docs/sprite.md

## Generating Sprites

> A tool that can generate sprites from svg files can be found in here: [spreet](https://github.com/flother/spreet)

Reviewer: ```suggestion
There are tools that can generate sprites from SVG files, such as [spreet](https://github.com/flother/spreet) and [spritezero](https://www.npmjs.com/package/@elastic/spritezero).
```
